hub/util/exceptions.py

```diff
@@ -65,6 +65,80 @@ def __init__(self):
         super().__init__("Ensure that len(size_list) + 1 == len(provider_list)")
 
 
+class LoginException:
+    def __init__(
+        self,
+        message="Error while logging in, invalid auth token. Please try logging in again.",
+    ):
+        super().__init__(message)
+
+
+# Exceptions encountered while interection with the Hub backend
+class AuthenticationException(Exception):
+    def __init__(self, message="Authentication failed. Please try logging in again."):
+        super().__init__(message)
+
+
+class AuthorizationException(Exception):
+    def __init__(
+        self,
+        message="You are not authorized to access this resource on Activeloop Server.",
+    ):
+        super().__init__(message)
+
+
+class ResourceNotFoundException(Exception):
+    def __init__(
+        self,
+        message="The resource you are looking for was not found. Check if the name or id is correct.",
+    ):
+        super().__init__(message)
+
+
+class BadRequestException(Exception):
+    def __init__(self, message):
+        message = f"One or more request parameters is incorrect\n{message}"
+        super().__init__(message)
+
+
+class OverLimitException(Exception):
+    def __init__(
+        self,
+        message="You are over the allowed limits for this operation. Consider upgrading your account.",
```

Comment:
Please remove "Consider upgrading your account."

Not sure if this can be done for all OverLimitException